appengine-skeleton-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -27,15 +27,16 @@
         <include>**/*.launch</include>
       </includes>
     </fileSet>
-    <fileSet encoding="UTF-8">
+    <fileSet filtered="true" encoding="UTF-8">
       <directory></directory>
       <includes>
-        <include>nbactions.xml</include>
+        <include>pom.xml</include>
```

Comment:
Hi, I don't think you need to have pom.xml explicitly. It worked properly without having this entry right?